master.cfg

```diff
@@ -290,7 +317,7 @@ c['status'].append(
     html.WebStatus(
         http_port=8010,
         authz=authz_cfg,
-        revlink=("%s%s" % (config['BASEURL'], config['REVLINKSTUB'])),))
+        revlink=("%s%s%s" % (config['BASE_URL'], config['OLA_REPO'], config['REVLINK_STUB'])),))
```

Comment:
It seems to be mostly fine with the default automatic one, apart from force builds, which currently have no properties, so we couldn't fix anyway. I may have a fix for that though.